env/src/test/java/com/sap/cloud/security/config/cf/CFEnvParserXsuaaTest.java

```diff
@@ -35,43 +35,43 @@ public CFEnvParserXsuaaTest() throws IOException {
 
 	@Test
 	public void getClientId() {
-		assertThat(cut.getClientId()).isEqualTo("clientId");
+		Assertions.assertThat(cut.getClientId()).isEqualTo("clientId");
```

Comment:
this seems to be a merge error and I do not expect any change on existing classes here in this PR beside of Environments.